Jenkinsfile

```diff
@@ -45,14 +52,24 @@ node('java') {
         // Once we've built, archive the artifacts and the test results.
         stage('Archive Artifacts / Test Results') {
             archiveArtifacts artifacts: '**/target/*.jar, **/target/*.war, **/target/*.hpi',
-                        fingerprint: true
+                    fingerprint: true
             if (runTests) {
                 junit healthScaleFactor: 20.0, testResults: '**/target/surefire-reports/*.xml'
             }
         }
     }
 }
 
+parallel linux: {
+    node('linux') {
```

Comment:
@slide I think this is actually clearer to use `linux` (I initially used `java`) since the [documentation](https://github.com/jenkins-infra/documentation/blob/master/ci.adoc) says indeed 

> `linux` : A Linux (Ubuntu 14.04 LTS) instance (alias of `java`)